ColabTool/LinkPost/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { PropTypes } from 'react';
+import {
+  Card,
+  Image,
+  LinkifiedText,
+  Text
+} from '@bufferapp/components';
+import style from './style.css';
+import Post from '../Shared/Post';
+
+// TODO: Add color={'gray'} to Text component in buffer-components then use in url below
+const LinkPost = ({
+  isConfirmingDelete,
+  isDeleting,
+  isWorking,
+  hovered,
+  imageSrc,
+  links,
+  linkAttachment,
+  manager,
+  onApproveClick,
+  onCancelConfirmClick,
+  onDeleteClick,
+  onDeleteConfirmClick,
+  onEditClick,
+  onMouseEnter,
+  onMouseLeave,
+  profile,
+  text,
+}) => {
+  const children = (
+    <div className={style['post-content']}>
```

Comment:
Heya @stevemdixon 😄 Thank you for taking a look! Accessing the className from `style` here is the same as `style.post-content` but dot-notation doesn’t let you use hyphens - so if it’s `style.some_class`  or `style.someClass` it’s fine but if it’s `some-class` then you have to use bracket-notation. 

Hope I understood your question right, here. 😄